src/app/pages/services/components/service-smb/service-smb.component.ts

```diff
@@ -220,16 +291,19 @@ export class ServiceSMBComponent {
     protected idmapService: IdmapService, protected userService: UserService,
     protected loader: AppLoaderService, protected dialog: MatDialog) {}
 
-  afterInit(entityEdit: any) {
+  afterInit(entityEdit: EntityFormComponent) {
+    entityEdit.submitFunction = body => {
+      delete body.idmap_tdb_range_high;
+      delete body.idmap_tdb_range_low;
+      return this.ws.call('smb.update', [body])
+    };
+
     this.entityEdit = entityEdit;
-    this.rest.get('services/cifs', {}).subscribe((res) => {
```

Comment:
I'm not exactly sure why this was here. The response doesn't seem to be used anywhere :man_shrugging:

Reply:
not sure either lol

Reply:
not sure either lol, does this all look right @anodos325 ?